### DISCUSSION TITLE
No tricks, all treats 🍬!   [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#36808] [New to GitHub]
This check-in is just like a bowl of candy - super sweet and chock full of goodness! Grab a piece or even a handful 🍫.

🌟**Community Superstars:** A round of applause for these community citizens 👏
* @Kevbrinx: For asking if GitHub could be used [beyond coding](https://github.com/orgs/community/discussions/35747), which was a fantastic reminder that there are so many use cases for GitHub besides software development! We can’t wait to see the comic you’re creating 👀.  
* @ZedZipDev and @ihnatenkoo: Worked together to solve @ZedZipDev’s challenge. There was a bit of back and forth, and great collaboration between these two on [this thread](https://github.com/orgs/community/discussions/35313) 🙏. 
* @adimael: answered a common [question](https://github.com/orgs/community/discussions/35628) in a concise and easy-to-navigate manner - thanks 🤩! 

🆕 **Launchpad**: Check out these collections for curated resources on getting started on GitHub 🚀.  
* **Learn to Code**: [This collection](https://github.com/collections/learn-to-code) has code snippets, projects to learn, and more ➕! 
* **How to choose (and contribute to) your first open source project**: The fun of open source is contributing to others, this [collection](https://github.com/collections/choosing-projects) has impactful projects for you to participate in 💃! 
* **Tools for Open Source**: [A list of softwares](https://github.com/collections/tools-for-open-source) to make running your open source project a little bit smoother 🧰.

🆒 **Cool stuff™️** happening around GitHub and the Community
* **Universe 2022 Agenda is Live**: Get your calendar 🗓️ out! Start planning which sessions you’ll attend during this year’s [Universe agenda ](https://github.blog/2022-10-11-the-github-universe-2022-agenda-is-live/) from @lclindeman. 
* **GitHub at United Nations General Assembly** 🌍: @malakumar85 [recaps](https://github.blog/2022-10-17-github-at-the-77th-united-nations-general-assembly/) how GitHub participated in the 77th UN General Assembly. 
* **GitHub Skill** 🆙: Expand your GitHub skillset each week with @ossanna16, [this week](https://github.com/orgs/community/discussions/36544) work on Issue Task Lists and labels 🏷️. 

Hope these treats didn’t give you too much of a sugar high. See you in two weeks for more GitHub sweetness 😋.

**Comment by adimael** [comment]
you're welcome :)

**Reply by queenofcorgis** [reply]
Rockstar 🎸 !

**Comment by ghost** [comment]
Can't wait to show you :)

**Reply by queenofcorgis** [reply]
YAAAAAS! 👀

**Comment by macbugs** [comment]
> **_"a fantastic reminder that there are so many use cases for GitHub besides software development"_**

- [Possibly the best github overview ever](https://www.reddit.com/r/github/comments/bc7p9s/comment/ekqwiwl)

> **_GitHub at United Nations General Assembly_**

...which is good to see when censorship remains an obstacle for many developers & academics.

- https://en.wikipedia.org/wiki/Censorship_of_GitHub
- https://github.com/github/gov-takedowns
- http://www.chinafirewalltest.com/?siteurl=github.com
